test/TCKOSnapshotTest.sol

```diff
@@ -157,4 +157,50 @@ contract TCKOSnapshotTest is Test {
             balance = balance - amount;
         }
     }
+
+    function testSnapshotValuesArePreserved() public {
+        uint256 balance = 250_000e6;
```

Comment:
In my opinion it would be better if the balance is taken from a wallet considering that there might be a change in setUp() however it is not a big deal.